tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -280,6 +280,11 @@ export const getMetricsSettingOverrides = createSelector(
   }
 );
 
+export const getMetricCardMaxWidth = createSelector(
```

Comment:
Should this be getMetric[s]CardMaxWidth to be consistent with the other ones?

Reply:
good catch. done